build.gradle

```diff
@@ -31,4 +37,21 @@ dependencyCheck {
 
 dependencyLocking {
     lockAllConfigurations()
+}
+
+licenseReport {
```

Comment:
Does this plugin work for subproject when you apply it to the root project? Could you please check that?